broker/api.py

```diff
@@ -382,3 +341,40 @@ def unbind(
         **kwargs,
     ) -> UnbindSpec:
         pass
+
+
+def parse_cookie_options(params):
+    forward_cookies = params.get("forward_cookies", None)
+    if forward_cookies is not None:
+        forward_cookies = forward_cookies.replace(" ", "")
+        if forward_cookies == "":
+            forward_cookie_policy = CDNServiceInstance.ForwardCookiePolicy.NONE.value
+            forwarded_cookies = []
+        elif forward_cookies == "*":
+            forward_cookie_policy = CDNServiceInstance.ForwardCookiePolicy.ALL.value
+            forwarded_cookies = []
+        else:
+            forward_cookie_policy = (
+                CDNServiceInstance.ForwardCookiePolicy.WHITELIST.value
+            )
+            forwarded_cookies = forward_cookies.split(",")
```

Comment:
Would it make sense to set `forwarded_cookies = []` at the start of the method (along with `forward_cookies`) and only modify if need be, which only appears to be in this particular case?  It looks like it's needed and used regardless all the way up to the `return` statement.

Reply:
I think I like this way better, as this is basically a set of finite options (all, none, allowlist) and not building a list of features

Reply:
Fair enough!